### PR TITLE
Livelab 912 Setup JMS

### PR DESCRIPTION
**Pull request by baojinlee-oracle**
# WMS 912

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [x] All filenames are lower case (including folders, images, files, etc.)
- [x] Filenames are descriptive
- [x] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [x] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [x] Image references in markdown contain an alternate text

### CODE REVIEW

File: java-management/setup-a-fleet/setup-a-fleet.md [hunk 1]
```diff
@@ -30,44 +30,106 @@ In this lab, you will:
...
+  There are 4 different advanced features available:
+  * Lifecycle management (LCM) - Manage the lifecycle of Java runtimes in your fleet by installing or removing reported Java runtime.
+  * Advanced usage tracking - Gain advanced insights into your Java workloads in the fleet by tracking application server and OpenJDK usage and identifying third party libraries used by applications.
+  * Crypto event analysis - Assess the impact of Oracle JRE and JDK Cryptographic roadmap on the applications running in your fleet.
+  * Java Flight Recorder (JFR) - Collect information about events in the application running in your fleet using Java Flight Recorder (JFR), a tool for collecting diagnostic and profiling data about a running Java application.
+
+
+  To learn more about the advanced features, see [Perform Java Lifecycle Management with Java Management Service](https://apexapps.oracle.com/pls/apex/dbpm/r/livelabs/view-workshop?wid=3202).
```
Comment (sktruong-oracle) [inline java-management/setup-a-fleet/setup-a-fleet.md:48]: should this be the new workshop name?

**Review by sktruong-oracle — APPROVED** [review]
lgtm

**Comment by baojinlee-oracle** [comment]
Modified the name of livelab 2 in introduction.

**Comment by baojinlee-oracle** [comment]
Removed the references to free tier in prerequisite as required by self QA

**Pull request merged**